qa/automation/CareerPortal.Tests.Scenarios/3. Send_CV/Send_CV.feature

```diff
@@ -0,0 +1,148 @@
+    Then CV should be successfully submitted
+
+    Examples:
+        | name        | email                    | message     |
+        | Egor  Berdin| egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: Correct CV submission without message
+    Given user entered a <name>, <email>, attached SV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name       | email                    |
+        | Egor Berdin| egor.berdin@quantori.com |
+ 
+
+
+Scenario Outline: CV submission without Name
+    Given user entered an <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then under the field "Name" a warning appears "This field is required"
+
+    Examples:
+        | email                    |
+        | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission without Email
+    Given user entered a <name>, attached CV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then under the field "Email" a warning appears "This field is required"
+
+    Examples:
+        | name         |
+        | Egor  Berdin |
+
+
+
+Scenario Outline: CV submission with 501 characters long message
+    Given user entered a <name>, <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    And writed a message 501 characters long
+    When user clicks on a button "Send"
+    Then under the field "Message" a warning appears "The maximum number of characters is 500"
+
+    Examples:
+        | name         | email                    |
+        | Egor  Berdin | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission with 500 characters long message
+    Given user entered a <name>, <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    And writed a message 500 characters long
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name         | email                    |
+        | Egor  Berdin | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission without checked the checkbox
+    Given user entered a <name>, <email>, <message>, attached CV (PDF less than 10 MB)
+    When user clicks on a button "Send"
+    Then the message next to the checkbox will turn red
+
+    Examples:
+        | name         | email                    | message     |
+        | Egor  Berdin | egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: CV submission without CV
+    Given user entered a <name>, <email>, <message>, checked the checkbox
+    When user clicks on a button "Send"
+    Then the button message "Attach CV" will turn red
+
+    Examples:
+        | name         | email                    | message     |
+        | Egor  Berdin | egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: CV submission with CV in different formats
```

Comment:
You should separate your scenarios for the Positive and Negative.
Scenario Outline: CV can be sent with one of the following extensions: PDF / DOC / DOCX / RTF /
And
Scenario Outline: CV can't be sent with invalid file extensions
At the result of this scenario you should see Warning window: Something went wrong
Invalid file format. Please try PDF, DOC, DOCX or RTF.